pyproject.toml

```diff
@@ -71,19 +69,16 @@ dependencies = [
     "toml==0.10.2",
     "tomli==2.2.1",
     "tqdm==4.67.1",
-    "typing-extensions==4.7.1",
+    "typing-extensions>=4.14.0",
```

Comment:
most other dependencies in this list are an exact version. are you sure you want to do `>=4.14.0` instead of say `==4.15.0` see the lockfile: it chose 4.15.0 instead of 4.14.0. if this library is causing issues to adding other deps it may be best to pin to a specific version to avoid that in the future as well

Reply:
I think that keeping it open like this will help avoid future issues. So if PA ever declares a dep that requires `4.16.0`, it will still work without me needing to update this again.